### PR TITLE
NOISSUE - Fix List Users When It is Admin

### PR DESCRIPTION
**Pull request by rodneyosodo**
### What does this do?
List all users when it is admin

### Which issue(s) does this PR fix/relate to?
Noissue

### List any changes that modify/break current functionality
No

### Have you included tests for your changes?
Yes

### Did you document any new/modified functionality?
No

### Notes
N/A

### CODE REVIEW

File: users/clients/service.go [hunk 1]
```diff
@@ -163,13 +163,24 @@ func (svc service) ListClients(ctx context.Context, token string, pm mfclients.P
 	if err != nil {
 		return mfclients.ClientsPage{}, err
 	}
+
+	// If the user is admin, fetch all things from database.
+	pol := policies.Policy{Subject: id, Object: clientsObjectKey, Actions: []string{listRelationKey}}
+	if err := svc.authorize(ctx, clientsObjectKey, pol); err == nil {
+		pm.Owner = ""
+		pm.SharedBy = ""
+		goto QUERY_DB
```
Comment (dborovcanin) [inline users/clients/service.go]: Avoid using labels and `goto`.

**Review by dborovcanin — CHANGES_REQUESTED** [review]

File: users/clients/service.go [hunk 2]
```diff
@@ -163,13 +163,23 @@ func (svc service) ListClients(ctx context.Context, token string, pm mfclients.P
...
 	if pm.SharedBy == MyKey {
 		pm.SharedBy = id
 	}
 	if pm.Owner == MyKey {
 		pm.Owner = id
 	}
 	pm.Action = "c_list"
+
+	// If the user is admin, fetch all things from database.
+	pol := policies.Policy{Subject: id, Object: clientsObjectKey, Actions: []string{listRelationKey}}
```
Comment (dborovcanin) [inline users/clients/service.go]: Use `p` instead of `pol`. Does this mean that we disable filtering by, say, owner for admin?

**Review by dborovcanin — CHANGES_REQUESTED** [review]

File: users/clients/service.go (hunk 2 above)
Comment (rodneyosodo) [reply users/clients/service.go]: Currently we don't filter by `owner`. This is because visibility changes the owner field

Comment (dborovcanin) [reply users/clients/service.go]: We need to provide that filter in the future. Please open a ticket for that.

**Review by dborovcanin — APPROVED** [review]

**Pull request merged**